mithril-aggregator/src/runtime/runner.rs

```diff
@@ -1236,4 +1285,94 @@ pub mod tests {
         assert!(!signed_entities.is_empty());
         assert!(!signed_entities.contains(&SignedEntityTypeDiscriminants::CardanoTransactions));
     }
+
+    #[tokio::test]
+    async fn is_outdated_return_false_when_message_is_not_expired_and_has_same_signed_entity_type()
+    {
+        assert!(!is_outdated_returned_when(IsExpired::No, true).await);
+    }
+
+    #[tokio::test]
+    async fn is_outdated_return_true_when_message_is_expired_and_has_same_signed_entity_type() {
+        assert!(is_outdated_returned_when(IsExpired::Yes, true).await);
+    }
+
+    #[tokio::test]
+    async fn is_outdated_return_true_when_message_is_not_expired_and_has_not_the_same_signed_entity_type(
+    ) {
+        assert!(is_outdated_returned_when(IsExpired::No, false).await);
+    }
+
+    #[tokio::test]
+    async fn is_outdated_return_true_when_message_is_expired_and_has_not_the_same_signed_entity_type(
+    ) {
+        assert!(is_outdated_returned_when(IsExpired::Yes, false).await);
+    }
```

Comment:
I am a bit confused about these tests which are not very clear: the `same_signed_entity_type` semantic should be replaced by `exists_newer_open_message`.

Reply:
Done ✅